apps/tng-dictionary/src/components/Term/Term.tsx

```diff
@@ -1,98 +1,107 @@
+import { motion } from 'framer-motion'
+import VolumeUpTwoToneIcon from '@mui/icons-material/VolumeUpTwoTone'
+import ErrorTwoToneIcon from '@mui/icons-material/ErrorTwoTone'
 
 // TODO move this to shared interfaces lib
 export type TermData = {
-  id: string;
+    id: string
 
-  contributor: string;
+    contributor: string
 
-  term: string;
+    term: string
 
-  termEnglish?: string;
+    termEnglish?: string
 
-  audioURL?: string;
+    audioURL?: string
 
-  sourceProject?: string;
+    sourceProject?: string
 }
 
 /* eslint-disable-next-line */
 export interface TermsDetailComponentProps {
-  termData?: TermData
+    termData?: TermData
 }
 
 export function Term(props: TermsDetailComponentProps) {
-  const audioRef = React.useRef();
-
-  const { termData } = props;
-
-  if (!termData) return (
-    <div>
-      <motion.div initial={{ opacity: 0 }} animate={{ opacity: 1 }} exit={{ opacity: 0 }} transition={{ duration: 1 }}>
-        <div className='termNot'>
-          <h1> Term not found <ErrorTwoToneIcon /></h1>
-          <h2>Lha ts'egwediʔal</h2>
-          <h2>"One couldn't find the word."</h2>
-        </div>
-      </motion.div>
-    </div>
-  );
-
-  const {
-    id, contributor, term, termEnglish, audioURL
-  } = termData;
-
-  /* AUDIO PLAYER */
-  let audio = new Audio(`${audioURL}`)
-
-  const start = () => {
-    audio.play().catch(console.log)
-  }
-
-  return (
-
-    <div>
-      <motion.div initial={{ opacity: 0 }} animate={{ opacity: 1 }} exit={{ opacity: 0 }} transition={{ duration: 1 }}>
-        <Typography className='termTitle' variant='h5'>
-          <b>{term}</b>
-          <motion.div
-            whileHover={{ scale: 1.2, }}
-            whileTap={{ scale: 0.95 }}
-            className='motionDiv'
-          >
-            <VolumeUpTwoToneIcon style={play} key={audioURL} onClick={start} />
-          </motion.div>
-        </Typography>
-        {/* Don't add a div if there's no termEnglish */}
-        <Divider className='divider' />
-        <div className='termClass' >
-          <div className='termInfo'>English:&nbsp; </div> {termEnglish ? termEnglish : ''}
-        </div>
-        <div className='termClass'>
-          <div className='termInfo'>Contributor:&nbsp;</div>{` ${contributor}`}
-        </div>
-        <div className='termClass'>
-          {`Term ID: ${id}`}
-        </div>
+    const audioRef = React.useRef()
+
+    const { termData } = props
+
+    if (!termData)
+        return (
+            <div>
+                <motion.div
+                    initial={{ opacity: 0 }}
+                    animate={{ opacity: 1 }}
+                    exit={{ opacity: 0 }}
+                    transition={{ duration: 1 }}
+                >
+                    <div className="termNot">
+                        <h1>
+                            {' '}
+                            Term not found <ErrorTwoToneIcon />
+                        </h1>
+                        <h2>Lha ts'egwediʔal</h2>
```

Comment:
It's ok for now, but we should think about how to pull such things from a config or the db.